server/tox.ini

```diff
@@ -54,6 +54,14 @@ deps =
 commands =
     mkdocs serve {posargs}
 
+[testenv:buildcli]
+extra =
+    dev
```

Comment:
If we're packing the non-dev cli, there's no reason to include the dev requirements. This also brings the question if you should be able to build just one or both. `tox` can probably have a switch flag to choose. Alternatively, we can have two different tox options (`buildcli` and `buildcli_dev`).

Reply:
Should only be able to build the non-dev

Reply:
Got it. Then you can just remove the 

```ini
extra =
     dev
```

This won't make a difference in the pyinstaller output, but by doing this `tox` won't need to install numpy and opencv just to setup the environment (which isn't needed for creating the non-dev cli). No effect on functionality, but will save some time.